network.go

```diff
@@ -79,6 +89,13 @@ func GetIP(r *http.Request, options ...Options) net.IP {
 	return net.ParseIP(host)
 }
 
+// GetSub returns sub from request JWT.
+func GetSub(r *http.Request, secret string) (string, error) {
```

Comment:
Should you name this function `GetJWTSub` ?
To have consistency with the method you have declared on limiter struct ?